### PR TITLE
Add Blockchain Explorer Search Extension

### PR DESCRIPTION
**Pull request by Namaskar-1F64F**
## Description

I added a new extension! It tries to automatically detect what is either in the clipboard or searched for in the List search, and returns a link to the corresponding blockchain explorer site. All of the sites share a similar structure, so switching between the different types is only a URL change away.

## Screencast

[screencast](https://cln.sh/ZLrCcb)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Namaskar-1F64F** [comment]
Ouch - screenshot size didn't make sense because the interface was too small for the 2000x1250 resolution. Looking into this 👀

File: extensions/blockchain-explorer-search/src/matchers.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  get parsedSearch() {
+    if (this.search.length === 64 && !this.search.startsWith("0x")) {
+      return "0x" + this.search;
+    }
+    return this.search;
+  }
+  match() {
+    if (!super.match()) return false;
+    if (this.tokenList) {
+      const foundToken = this.tokenList.find(({ address }) => address === this.search);
```
Comment (Geczy) [inline extensions/blockchain-explorer-search/src/matchers.ts:100]: can you do search ignore case matching?

**Comment by RusseII** [comment]
Any updates on this? Really excited for this extension!

File: extensions/blockchain-explorer-search/src/matchers.ts (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/blockchain-explorer-search/src/matchers.ts:100]: Can you look into this @Namaskar-1F64F

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @Namaskar-1F64F 👋 

Thanks for the extension, I tested it and I only have minor feedback:

The search command, when you have something in the clipboard suffer from [Empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states) - could you look into this? Maybe add a loading state that you can pass is isLoading={} to your list 😄 

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/blockchain-explorer-search/src/matchers.ts (hunk 1 above)
Comment (Namaskar-1F64F) [reply extensions/blockchain-explorer-search/src/matchers.ts]: Good suggestion 👍 Implemented.

File: extensions/blockchain-explorer-search/src/search.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ export default function Command() {
       accessories={clipboard === search ? [{ text: "From clipboard", icon: Icon.Clipboard }] : []}
       actions={
         <ActionPanel>
-          <Action.OpenInBrowser title="Open link in Browser" url={match.path} />
+          <Action.OpenInBrowser url={match.path} />
```
[the remark this answers is not in the document]
Comment (Namaskar-1F64F) [reply extensions/blockchain-explorer-search/src/search.tsx:75]: `match.path` is the link, and open in browser is the first action in the action panel (bottom right). command + k will show the other options, like to create a quick link

**Comment by Namaskar-1F64F** [comment]
No option to re-request a review. Am I missing something or GH bug?
CleanShot 2022-06-09 at 18 15 12@2x

Anyway, I fixed the search problem. Tested by clearing localstorage. 

@pernielsentikaer

**Comment by Geczy** [comment]
> No option to re-request a review. Am I missing something or GH bug? CleanShot 2022-06-09 at 18 15 12@2x
> 
> Anyway, I fixed the search problem. Tested by clearing localstorage.
> 
> @pernielsentikaer

i think its because of this sequence of events. @pernielsentikaer has not requested changes since

2022-06-09 at 11 18 40@2x

File: extensions/blockchain-explorer-search/src/search.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pernielsentikaer) [reply extensions/blockchain-explorer-search/src/search.tsx:75]: There was a wrong naming convention here, it should be title cased - the title is also automatically added in `Action.OpenInBrowser` if none provided

**Review by pernielsentikaer — APPROVED** [review]
I think we're ready to merge 🎉 

In the next version, it could be good to look into some of the logos' - black text in dark appearance is not the best 😄

**Pull request merged**